homeassistant/components/switch/rachio.py

```diff
@@ -19,7 +19,7 @@
 CONF_MANUAL_RUN_MINS = 'manual_run_mins'
 DEFAULT_MANUAL_RUN_MINS = 10
 
-MIN_UPDATE_INTERVAL = timedelta(minutes=5)
+MIN_UPDATE_INTERVAL = timedelta(seconds=5)
```

Comment:
@balloob, I've been running mine with a 30 second interval and it seems to have the same desired effect, in that regardless of 5 or 30 seconds it updates the correct state on the frontend after about 30 seconds have passed.  Do you think leaving timedelta at 30 seconds would be acceptable to not overload the API?